onedal/linear_model/linear_model.py

```diff
@@ -14,28 +14,31 @@
 # limitations under the License.
 # ==============================================================================
 
-from abc import ABCMeta, abstractmethod
+from abc import ABC, abstractmethod
 from numbers import Number
 
 import numpy as np
 
 from daal4py.sklearn._utils import get_dtype, make2d
+from onedal import _backend
 
-from ..common._base import BaseEstimator as onedal_BaseEstimator
 from ..common._estimator_checks import _check_is_fitted
-from ..common._mixin import RegressorMixin
+from ..common._policy import _get_policy
 from ..common.hyperparameters import get_hyperparameters
 from ..datatypes import _convert_to_supported, from_table, to_table
 from ..utils import _check_array, _check_n_features, _check_X_y, _num_features
 
 
-class BaseLinearRegression(onedal_BaseEstimator, metaclass=ABCMeta):
+class BaseLinearRegression(ABC):
```

Comment:
Why are you removing onedal_BaseEstimator?
IT should be consistent with other estimators